### PR TITLE
Add initial settings for internationalisation

### PR DESCRIPTION
**Pull request by OnaMosimege**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -120,3 +122,13 @@ docker-shell:
 check:
 	clear
 	@docker-compose run --rm web python manage.py check
+
+make-messages:
+	clear
+	@docker-compose run --rm web python manage.py makemessages -l en
+	@docker-compose run --rm web python manage.py makemessages -l af
```
Comment (friedelwolff) [inline Makefile:129]: Such a list will become unwieldy quickly. Does `--all` work well?

File: Makefile [hunk 2]
```diff
@@ -120,3 +122,13 @@ docker-shell:
...
+
+compile-messages:
+	clear
+	@docker-compose run --rm web python manage.py compilemessages -l en
+	@docker-compose run --rm web python manage.py compilemessages -l af
```
Comment (friedelwolff) [inline Makefile:134]: Same here. According to `compilemssages --help`  compilemessages processes all files by default. Maybe good enough for us?

File: app/app/settings.py [hunk 3]
```diff
@@ -173,6 +174,21 @@
...
 LOGGING_FOLDER_DEFAULT = os.path.abspath(os.path.join("/logging/"))
 
+# Internationalization
+
+USE_I18N = True
+
+LANGUAGES = [
+    ("en", "English"),
+    ("af", "Afrikaans"),
+]
```
Comment (friedelwolff) [inline app/app/settings.py:184]: For maintainability, please keep it alphabetically sorted by language code.

**Review by friedelwolff — CHANGES_REQUESTED** [review]
Thank you for this. Please check that we don't commit .mo files by adding an entry to `.gitignore`. Generated files should ideally not be in version control files, and if we generate this in our Docker image, that should be sufficient.

File: app/app/settings.py (hunk 3 above)
Comment (OnaMosimege) [reply app/app/settings.py:184]: will do

File: Makefile (hunk 1 above)
Comment (OnaMosimege) [reply Makefile:129]: I tried --all, does not seem to generate anything

File: Makefile (hunk 2 above)
Comment (OnaMosimege) [reply Makefile:134]: Okay, will try this. Thanks

**Review by friedelwolff — CHANGES_REQUESTED** [review]
Thanks for the update. Please remove the .mo files from the commit. We basically never want generated files committed in the version control system.

**Review by friedelwolff — APPROVED** [review]

**Pull request merged**